ocrd_segment/ocrd-tool.json

```diff
@@ -291,6 +291,27 @@
           "default": "",
           "description": "Comma-separated list of forbidden image features (e.g. `binarized,despeckled`)."
         },
+        "classes": {
+          "type": "string",
+          "enum": [
```

Comment:
```suggestion
          "pattern": "((Noise|LineDrawing|Advert|Image|Chart|Music|Graphic|Unknown|Custom|Separator|Maths|Text|Map|Chem|Table)Region(,(Noise|LineDrawing|Advert|Image|Chart|Music|Graphic|Unknown|Custom|Separator|Maths|Text|Map|Chem|Table)Region)*)?" 
```

:)

Reply:
Yes, `pattern` is generally enforced. Whether it's sensible is another question, the resulting error message won't be very intuitive for the user.